nuxeo-retention-web/elements/nuxeo-retention-menu.js

```diff
@@ -49,6 +49,8 @@ class RetentionMenu extends mixinBehaviors([FiltersBehavior, I18nBehavior, Routi
         </nuxeo-menu-item>
         <nuxeo-menu-item label="retention.search" name="search" route="page:retentionSearch"> </nuxeo-menu-item>
         <nuxeo-menu-item label="retention.events" name="events" route="page:retentionEvents"> </nuxeo-menu-item>
+        <nuxeo-menu-item label="retention.analytics" name="analytics" route="page:retentionAnalytics">
```

Comment:
As I explained to you, I observed that the analytic screen is not refreshed, when we click on the tab menu. Even with a new content/new documents/new retention rule. To see the new things, we have to change the dates

Reply:
thanks, yes, I need to investigate as this is a tricky issue